src/Core/Components/Menu/FluentMenuItem.razor

```diff
@@ -0,0 +1,36 @@
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
+@using Microsoft.FluentUI.AspNetCore.Components.Extensions
+@inherits FluentComponentBase
+<fluent-menu-item id="@Id"
+                  class="@ClassValue"
+                  style="@StyleValue"
+                  role="@Role.ToAttributeValue()"
+                  checked="@Checked"
+                  disabled="@Disabled"
+                  @attributes="AdditionalAttributes"
+                  @onmenuitemchange="@OnChangeHandlerAsync">
+    @if (IconIndicator != null)
+    {
+        <FluentIcon Value="@IconIndicator" Slot="@((ChildContent is null && Label is null) ? null : "indicator")" />
```

Comment:
That could be better (easier to read) to have a private property like this `private bool EmptyContent => ChildContent is null && string.IsNullOrEmpty(Label);`.